src/components/titles/index.js

```diff
@@ -14,6 +14,7 @@ var interactConstants = require('../../constants/interactions');
 
 var OPPOSITE_SIDE = require('../../constants/alignment').OPPOSITE_SIDE;
 var numStripRE = / [XY][0-9]* /;
+var MATHJAX_PADDING_BOTTOM = 10;
```

Comment:
Should this depend on on title font size?

Also I'm wondering if you add any spacing (or zero?) between title and subtitle for normal text we could have a constant for it with a TODO comment to help expose it later.

Reply:
Yes I suppose it should depend on the subtitle font size (since Mathjax size is not affected by font size as far as I can tell).

Reply:
> Also I'm wondering if you add any spacing (or zero?) between title and subtitle for normal text we could have a constant for it with a TODO comment to help expose it later.

Sure, I can add a constant. I don't see a need for adding any spacing at the moment.